src/include/cache.h

```diff
@@ -199,18 +197,6 @@ struct __wt_cache {
     uint64_t las_insert_count; /* Count of inserts to lookaside */
     uint64_t las_remove_count; /* Count of removes from lookaside */
 
-    bool las_reader; /* Indicate an LAS reader to sweep */
-    WT_RWLOCK las_sweepwalk_lock;
-    WT_SPINLOCK las_sweep_lock;
-    WT_ITEM las_sweep_key;            /* Track sweep position. */
-    uint32_t las_sweep_dropmin;       /* Minimum btree ID in current set. */
-    uint8_t *las_sweep_dropmap;       /* Bitmap of dropped btree IDs. */
-    uint32_t las_sweep_dropmax;       /* Maximum btree ID in current set. */
-    uint32_t las_max_btree_id;        /* Maximum btree ID for sweep. */
-    WT_ITEM las_max_key;              /* Maximum key for sweep. */
-    wt_timestamp_t las_max_timestamp; /* Maximum timestamp for sweep. */
-    uint64_t las_max_txnid;           /* Maximum txn ID for sweep. */
-
     uint32_t *las_dropped;    /* List of dropped btree IDs. */
```

Comment:
Thanks, @kommiharibabu, I've removed references to the dropped btrees.  Please take another look.  One question, I just want to confirm that I would be merging into pm-1521-durable-history when this is approved?

Reply:
Thanks for the changes @ddanderson. Yes, this change will go into `pm-1521-durable-history` branch. That is the feature branch for `durable history` project.

Reply:
Just curious why that branch is named `pm-1521-...`? That is the first and only branch ever to be named for a PM ticket and the merge will go into the Product Management ticket instead of a WiredTiger ticket. That means later searching in JIRA of the WiredTiger ticket space will not find it. Is this a new process I'm not familiar with?

Reply:
`pm-1521-durable-history` is the feature branch of durable history. All the code changes that are done for the durable history, `pm-1521-...` is the final branch. Once all the tests pass, this branch will get merged into `develop`. It is just the branch name that is named with a PM ticket. All the work done with WT ticket changes still represents inside their tickets itself. There is no change in the process, it is just the final branch that the code gets merged is different.

Reply:
It is not just a branch name. There are github hooks that push all changes/commits into the ticket that is the prefix of the branch. So every change that has been merged into that branch, including all sub-branches, will be sent to the Jira ticket PM-1521. That is a change in process because no other PM ticket has all project commits pushed to it. Maybe that's okay and the direction that Product management, the owner of the PM ticket space, wants to go.

You are correct that the individual sub-branch merges should go to their tickets. But it is not "just" a branch name. It will have an effect when merged with develop.

Reply:
Note - the code changes for this discussion are resolved.  But I'm leaving this open for now, I think the discussion about branch naming has not been resolved.